addons/web_tour/static/src/js/tip.js

```diff
@@ -314,36 +330,58 @@ var Tip = Widget.extend({
         this.$el.addClass("o_animated");
     },
     _bind_anchor_events: function () {
+        this.$bubbleEventAnchors = this.$anchor;
```

Comment:
I wouldn't open the tip with the alt anchor, but maybe @odoongr could give its opinion on this. If we don't, this variable should be rename `this.$bubbleEventAnchor`

Reply:
@aab-odoo @IT-Ideas I agree that hovering the alt trigger should not open the tip